gui/base.py

```diff
@@ -346,6 +351,18 @@ def _render_footer_links(self) -> None:
                     "width: 20px; height: 20px; fill: currentColor"
                 )
                 ui.tooltip("Follow us on Instagram")
+            
+            # Linkdin button
+            linkdin_btn = ui.button(
+                color="accent",
+                on_click=lambda: self.navigate_to_external(gui_urls.linkdlin_url),
+            ).props("flat round")
+            with linkdin_btn:
+                linkdin_svg = self._load_svg_icon("instagram")
```

Comment:
Shouldn't this be loading a LinkedIn logo? It's currently going to load "instagram" logo. To add an icon, you need to find an `.svg` of an icon, place it in `gui/icons/` and modify the `_load_csv_icon()` method accordingly.

I found insta/github icons at https://simpleicons.org which are [licensed by CC0](https://creativecommons.org/publicdomain/zero/1.0/). We should make sure that similar llicense is in place for any linked in icon too.